src/PrestaShopBundle/Resources/config/services/adapter/tax_rules_group.yml

```diff
@@ -2,6 +2,22 @@ services:
   _defaults:
     public: true
 
+  prestashop.adapter.tax_rules_group.command_handler.add_tax_rules_group_handler:
+    class: 'PrestaShop\PrestaShop\Adapter\TaxRulesGroup\CommandHandler\AddTaxRulesGroupHandler'
```

Comment:
```suggestion
  PrestaShop\PrestaShop\Adapter\TaxRulesGroup\CommandHandler\AddTaxRulesGroupHandler:
```

Reply:
We favor FQCN service definitions now, we didn't have yet the time to migrate all the definitions for existing services (which will need aliases) but for new definitions it's better to use it right away as it will avoid having to add a temporary name alias

Reply:
Fixed